.github/pull_request_template.md

```diff
@@ -1,9 +1,12 @@
-**Description:** <Describe what has changed.>
 <!--Ex. Fixing a bug - Describe the bug and how this fixes the issue.
 Ex. Adding a feature - Explain what this achieves.-->
+Description:
```

Comment:
I agree that it's nicer for the PR, but the reason I removed it was that every time we merge a PR, this gets automatically populated into the commit (see https://github.com/open-telemetry/opentelemetry-collector-contrib/commit/c3e750c4632c527f707bbb94529db929baef1bd7 as an example) which makes reading the git history slightly more difficult.

Reply:
Would converting these to [headings](https://www.markdownguide.org/basic-syntax/#headings) be more of a happy medium? Or do you think that would also be unclear in commits?

Reply:
I think headings would be an an improvement over bolding, even if we used the h4-equivalent headings.

Here's an example of what this might look like taken from a spec repo commit: https://github.com/open-telemetry/opentelemetry-specification/commit/074466bcad9a39711fd21e83b342ef6245acac08